modules/mcagar/src/main/ts/ephox/mcagar/keyboard/TypeText.ts

```diff
@@ -40,21 +44,17 @@ const insertCharAtSelection = (doc: Document, chr: string): void => {
 };
 
 const typeChar = (doc: SugarElement<Document>, chr: string): void => {
+  // TODO: Make this respect preventDefault on each keydown/keypress
   Keyboard.activeKeydown(doc, chr.charCodeAt(0), { });
   Keyboard.activeKeypress(doc, chr.charCodeAt(0), { });
-  insertCharAtSelection(doc.dom, chr);
+  insertCharAtSelection(doc, chr);
   Keyboard.activeKeyup(doc, chr.charCodeAt(0), { });
 };
 
 const typeContentAtSelection = (doc: SugarElement<Document>, text: string): void => {
   Arr.map(text.split(''), Fun.curry(typeChar, doc));
 };
 
-const sTypeContentAtSelection = <T>(doc: SugarElement<Document>, text: string): Step<T, T> => Step.sync(() => {
```

Comment:
Was this moved? I can't see it anywhere else. Or we don't use it anywhere?

Reply:
Sorry, should have mentioned that. It's no longer used, as core no longer uses the old Chain/Step tests.